apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -36,7 +41,24 @@ interface CounterProps {
 
 const styleQuestionCircle: CSSProperties = { color: colors.gray['5'], cursor: 'help' }
 
-const Counter: React.FC<CounterProps> = ({ label, value, type = 'success', hint, downloadButton, isLoading }) => (
+export const RecipientCounterContainer: React.FC<{ title: React.ReactNode }> = ({ children, title }) => {
+    return (
+        <Card>
+            <Typography.Text>
+                { title }
```

Comment:
we don't use extra spaces inside this file and others tsx files. Why do you want to add it here?